<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkTotalSegmentatorPreferencePage</class>
 <widget class="QWidget" name="QmitkTotalSegmentatorPreferencePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>779</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="invisibleLabel"/>
   </item>
   <item row="0" column="1" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="customEnvLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Custom Env. Path:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="deviceSpinLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Device:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="OverrideLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Use Custom Installation</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QPushButton" name="installTotalButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string>Install TotalSegmentator</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="ctkComboBox" name="deviceComboBox" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ctkCheckBox" name="overrideBox" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="checked" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="ctkComboBox" name="sysPythonComboBox" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="sysPythonlabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>System Python (&gt;=3.9):</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QPushButton" name="clearTotalbutton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string>Clear Install</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="ctkComboBox" name="customEnvComboBox" native="true"/>
     </item>
     <item row="5" column="0" colspan="4">
      <widget class="QLabel" name="statusLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
	  <item row="1" column="0">
		  <spacer name="verticalSpacer">
			  <property name="orientation">
				  <enum>Qt::Vertical</enum>
			  </property>
			  <property name="sizeHint" stdset="0">
				  <size>
					  <width>20</width>
					  <height>40</height>
				  </size>
			  </property>
		  </spacer>
	  </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkComboBox</class>
   <extends>QWidget</extends>
   <header>ctkComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCheckBox</class>
   <extends>QWidget</extends>
   <header>ctkCheckBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
